### DISCUSSION TITLE
A11y improvements

### DISCUSSION THREAD

**Discussion by flipvh** [element-plus/element-plus#1697] [Ideas]
Hi all,

We work with universities and for them, accessibility is increasingly important. We are considering getting actively involved with this for element-plus, but are interested what others think about this.

Some ideas:
-  Allow for font-size changes by browser by using 'rem' in css instead of 'px'.
-  Making it more easy to set high contrast version of default element theme
-  Keyboard navigation related improvements, perhaps a component for a hidden 'skip to content' button

Please let me know your thoughts!

**Comment by jw-foss** [comment]
Yes, a11y has always been the topic the team talks about. We planned launching `a11y improvement program` after the first stable version gets released. FYI, the team does't really have any expert on a11y, so that we could need the help from the community so that we can make this project accessible for all kinds of people. You might see some work has been done but it was just inadequate.
